packages/dashboard/src/components/journeys/edgeTypes/workflowEdge.tsx

```diff
@@ -103,6 +152,21 @@ export default function WorkflowEdge({
           +
         </text>
       </g>
+      {getLabelText().length > 0 && (
+        <EdgeLabelRenderer>
+          <div
```

Comment:
Can you provide this with a gray background, so that it's more readable when overlapping the edge? You can use the same color as the label text.

Reply:
Done 

<img width="411" alt="DF-316_updated" src="https://github.com/dittofeed/dittofeed/assets/64700961/5c69b3b1-e967-4956-a179-64c38bfcf4df">